maintenance/troubleshoot/commands.md

```diff
@@ -0,0 +1,499 @@
+
+```bash
+ip r | grep bird
+```
+
+```
+192.168.66.128/26 via 10.0.0.12 dev tunl0 proto bird onlink 
+192.168.180.192/26 via 10.0.0.10 dev tunl0 proto bird onlink 
+blackhole 192.168.231.0/26 proto bird 
+```
+
+#### Verify BIRD routing table
+
+**Note**: The BIRD routing table gets pushed to node routing tables.
+
+```bash
+kubectl exec -it -n calico-system calico-node-8cfc8 -- /bin/bash
+```
+
+```
+[root@ip-10-0-0-11 /]# birdcl
+BIRD v0.3.3+birdv1.6.8 ready.
+bird> show route
+0.0.0.0/0          via 10.0.0.1 on eth0 [kernel1 18:13:33] * (10)
+10.0.0.0/24        dev eth0 [direct1 18:13:32] * (240)
+10.0.0.1/32        dev eth0 [kernel1 18:13:33] * (10)
+10.48.231.2/32     dev calieb874a8ef0b [kernel1 18:13:41] * (10)
+10.48.231.1/32     dev caliaeaa173109d [kernel1 18:13:35] * (10)
+10.48.231.0/26     blackhole [static1 18:13:32] * (200)
+10.48.231.0/32     dev vxlan.calico [direct1 18:13:32] * (240)
+10.48.180.192/26   via 10.0.0.10 on eth0 [Mesh_10_0_0_10 18:13:34] * (100/0) [i]
+                   via 10.0.0.10 on eth0 [Mesh_10_0_0_12 18:13:41 from 10.0.0.12] (100/0) [i]
+                   via 10.0.0.10 on eth0 [kernel1 18:13:33] (10)
+10.48.66.128/26    via 10.0.0.12 on eth0 [Mesh_10_0_0_10 18:13:36 from 10.0.0.10] * (100/0) [i]
+                   via 10.0.0.12 on eth0 [Mesh_10_0_0_12 18:13:41] (100/0) [i]
+                   via 10.0.0.12 on eth0 [kernel1 18:13:36] (10)
+```
+
+#### Capture traffic
+
+For example, 
+
+```bash
+sudo tcpdump -i calicofac0017c3 icmp
+```
+
+### Network policy
+
+#### Verify existing Kubernetes network policies
+
+```bash
+kubectl get networkpolicy --all-namespaces
+```
+
+```
+NAMESPACE   NAME             POD-SELECTOR   AGE
+client      allow-ui         <none>         20m
+client      default-deny     <none>         4h51m
+stars       allow-ui         <none>         20m
+stars       backend-policy   role=backend   20m
+stars       default-deny     <none>         4h51m
+```
+
+#### Verify existing {{site.prodname}} network policies
+
+```bash
+calicoctl get networkpolicy --all-namespaces -o wide
+```
+
+```
+NAMESPACE     NAME                         ORDER   SELECTOR                                                       
+calico-demo   allow-busybox                50      app == 'porter'                                                
+client        knp.default.allow-ui         1000    projectcalico.org/orchestrator == 'k8s'                        
+client        knp.default.default-deny     1000    projectcalico.org/orchestrator == 'k8s'                        
+stars         knp.default.allow-ui         1000    projectcalico.org/orchestrator == 'k8s'                        
+stars         knp.default.backend-policy   1000    projectcalico.org/orchestrator == 'k8s' 
+stars         knp.default.default-deny     1000    projectcalico.org/orchestrator == 'k8s'                        
+```
+
+#### Verify existing Kubernetes global network policies
+
+```bash
+kubectl get globalnetworkpolicies 
```

Comment:
There's no such thing as a Kubernetes GlobalNetworkPolicy - this is going to return Calico GNPs, so we don't need both this section as well as the one below.

Reply:
Yeah! That was bogus entry for sure. Removed.